experimental/dds/attributable-map/api-extractor-cjs.json

```diff
@@ -0,0 +1,8 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "../../../common/build/build-common/api-extractor-base.cjs.primary.json",
+	// CJS is actually secondary; so, no report.
```

Comment:
Should we have a api-extractor-base.cjs.secondary.json config?

Reply:
That makes sense.  We're 50 packages in, so we'll need to do a cleanup pass.  Now tracked by AB#7417.